### PR TITLE
add try-runtime to ping pallet

### PR DESCRIPTION
**Pull request by brunopgalvao**
Fixes: https://github.com/paritytech/cumulus/issues/2704

### CODE REVIEW

File: parachains/pallets/ping/Cargo.toml [hunk 1]
```diff
@@ -30,3 +30,5 @@ std = [
 	"frame-system/std",
 	"xcm/std",
 ]
+
+try-runtime = ["frame-support/try-runtime"]
```
Comment (ggwpez) [inline parachains/pallets/ping/Cargo.toml:34]: `zepter lint propagate-feature --feature try-runtime -p cumulus-ping --fix` adds a few more:  

```suggestion
try-runtime = [
       "frame-support/try-runtime",
       "cumulus-pallet-xcm/try-runtime",
       "frame-system/try-runtime",
       "sp-runtime/try-runtime"
]
```

**Review by ggwpez — APPROVED** [review]
Not sure if we need this pallet at all…

**Comment by brunopgalvao** [comment]
> Not sure if we need this pallet at all…

Here is the original PR for the ping pallet (aka spambot): https://github.com/paritytech/cumulus/pull/391

Currently it is in the Trappist `stout` runtime: 
- https://github.com/paritytech/trappist/blob/235f8666b9eb1c4dc951a85b8c5c85d03b835345/runtime/stout/src/lib.rs#L550

That is why I need the try-runtime features for it.

I will let someone else comment if this pallet is still needed or not in Cumulus.

**Review by liamaharon — APPROVED** [review]

File: parachains/pallets/ping/Cargo.toml (hunk 1 above)
Comment (brunopgalvao) [reply parachains/pallets/ping/Cargo.toml]: Cool [tool](https://github.com/ggwpez/zepter), will be sure to add this to my arsenal.

**Comment by brunopgalvao** [comment]
bot merge

**Comment by brunopgalvao** [comment]
bot merge

**Pull request merged**